src/main/java/org/jenkinsci/plugins/DependencyCheck/transformer/FindingsTransformer.java

```diff
@@ -143,6 +151,9 @@ public JSONObject transform(List<Finding> findings) {
             row.put("dependency.md5", escape(dependency.getMd5()));
             row.put("dependency.sha1", escape(dependency.getSha1()));
             row.put("dependency.sha256", escape(dependency.getSha256()));
+            if (CollectionUtils.isNotEmpty(dependency.getProjectReferences())) {
+                row.put("dependency.projectReferences", escape(String.join(", ", dependency.getProjectReferences())));
```

Comment:
comma separated list is not clear. Please render as a ul list

Reply:
OK